app/src/main/java/org/thoughtcrime/securesms/ui/components/ButtonSize.kt

```diff
@@ -0,0 +1,40 @@
+package org.thoughtcrime.securesms.ui.components
+
+import android.annotation.SuppressLint
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.LocalMinimumInteractiveComponentEnforcement
+import androidx.compose.material.LocalTextStyle
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.thoughtcrime.securesms.ui.baseBold
+import org.thoughtcrime.securesms.ui.extraSmallBold
+
+interface ButtonSize {
+    @OptIn(ExperimentalMaterialApi::class)
+    @SuppressLint("ComposableNaming")
+    @Composable fun applyConstraints(content: @Composable () -> Unit) {
+        CompositionLocalProvider(
+            LocalMinimumInteractiveComponentEnforcement provides false,
+            LocalTextStyle provides textStyle,
```

Comment:
Have you tested this code? The buttons look unstyled to me.
I tried changing this to `LocalTextStyle provides h1,` and the buttons look the same.

Reply:
oh weird, I was suspecting it wasn't quite working, so I changed the colors of normal and slim buttons to different colors, and observed, at least that was working.

Reply:
Right, so `Button` does call some method that sets the Style. fixed.

Reply:
See new `@Preview` in `Button` to verify.